pkg/api/v1/conversion_generated.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	api "k8s.io/kubernetes/pkg/api"
 	resource "k8s.io/kubernetes/pkg/api/resource"
+	experimental "k8s.io/kubernetes/pkg/apis/experimental"
```

Comment:
This import seems quite wrong to me. Experimental conversions should go in the experimental package.